packages/mds-stream-processor/labelers/geography-labeler.ts

```diff
@@ -14,17 +14,33 @@
     limitations under the License.
  */
 
+import db from '@mds-core/mds-db'
 import { UUID, Nullable, Telemetry } from '@mds-core/mds-types'
+import { pointInShape, filterEmptyHelper } from '@mds-core/mds-utils'
 import { MessageLabeler } from './types'
 
 export interface GeographyLabel {
-  geography_id: UUID[]
+  geography_ids: UUID[]
 }
 
 export const GeographyLabeler: () => MessageLabeler<
   { telemetry?: Nullable<Telemetry> },
   GeographyLabel
 > = () => async ({ telemetry }) => {
-  // TODO: Add Point-in-polygon checks
-  return { geography_id: telemetry ? [] : [] }
+  const gps = telemetry?.gps
+  if (gps) {
+    const { lat, lng } = gps
+
+    const geographies = await db.readGeographies()
```

Comment:
This should probably be done just once and cached. I have an idea about how to allow some async initialization for processors.

Reply:
Geographies can be updated constantly, though.